test/localize.js

```diff
@@ -20,9 +20,10 @@ describe( 'localize()', function() {
 			render: emptyRender
 		} );
 
+		var myComponentName = MyComponent.displayName || MyComponent.name || '';
```

Comment:
Hi @yoavf, I think having this chain of fallbacks isn't ideal in a test since it creates some ambiguity around what myComponentName is and thus the final string being tested in .to.equal(...)
By the looks of this I don't think MyComponent has either a displayName or a name property so the expected value is ('Localized' + '').

My suggestion would be something like this:
    var MyComponent = React.createClass( {
        displayName: 'MyComponent',
        render: emptyRender
    } );
    ...
    expect( LocalizedComponent.displayName ).to.equal( 'LocalizedMyComponent');

It could be worth adding seperate tests to cover cases where no displayName is supplied but name is, or when neither displayName or name are supplied.